tests/unistd/links.c

```diff
@@ -82,5 +82,20 @@ int main() {
 #endif
   errno = 0;
 
+#ifdef NODEFS
```

Comment:
How about `ifndef WASMFS` since it seems like only wasmfs should be exempt.

In the the long run the all the FS implementations should do right thing though.  Which return value is correct?     Is WASMJS lacking right now?

@tlively @kripken ?

Reply:
I am pretty sure my test matches the behavior on native linux (I extracted it from the Python test suite).

Reply:
If you look at `man access` it says that it is supposed to chase symlinks:
> access() checks whether the calling process can access the file pathname.  If pathname is a symbolic link, it is dereferenced.
> ...
> access() and faccessat() shall fail if:
> ...
>       ELOOP  Too many symbolic links were encountered in resolving pathname.